apps/console/src/features/server-configurations/pages/connector-edit-page.tsx

```diff
@@ -419,7 +429,7 @@ export const ConnectorEditPage: FunctionComponent<ConnectorEditPageInterface> =
                         >
                             When self registration is enabled, users can register via the
                             <strong>
-                                Create an account
+                                Register
```

Comment:
Hope this is a unified experience?

Reply:
Yes, this is unified UX.

<img width="1501" alt="Screenshot 2024-03-24 at 13 20 24" src="https://github.com/wso2/identity-apps/assets/41533942/2d5e9965-5417-4c94-a43f-596bd76d1d82">